app/controllers/api/lists_controller.rb

```diff
@@ -21,17 +21,34 @@ def remove
     end
 
     def save
-      ingredients = current_api_user.kitchen_list.ingredients
-      params[:list][:ingredients].each do |ingredient|
+      params_ingredients = params[:list][:ingredients]
+      orig_ingredients = current_api_user.ingredients
```

Comment:
@nicholasblock78 Some time soon let's refactor the var names here to server_ingredients and client_ingredients, that might make it a little clear.